client/blocks/app-banner/docs/example.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import AppBanner from '../';
+import { PREFERENCE_NAME } from '../utils';
+import { savePreference } from 'state/preferences/actions';
+
+function AppBannerExample( clearPreference ) {
+	return (
+		<div className="docs__design-assets-group">
+			<h2>
+				<a href="/devdocs/blocks/app-banner">App Banner</a>
+			</h2>
+			<AppBanner userAgent="android" />
+			<Button onClick={ clearPreference }>Reset Dismiss Preference</Button>
+		</div>
+	);
+}
+
+const ConnectedAppBannerExample = connect(
+	null,
+	{ clearPreference: savePreference( PREFERENCE_NAME, null ) }
+)( AppBannerExample );
+
+ConnectedAppBannerExample.displayName = 'AppBannerExample';
```

Comment:
I didn't notice that, just carried it over since I noticed it in other docs examples. Also I remember vaguely that we had problems with component example titles in devDocs due to minification - is that still the case? 

Anyway, it's not relevant at this point since I removed this file completely for now.